### DISCUSSION TITLE
Empty video memory

### DISCUSSION THREAD

**Discussion by RetroAndDev** [CosmosOS/Cosmos#1709] [Q&A]
Hello, I would like to know if it is possible to empty the video memory because after several seconds the graphic interphase freezes. I use the graphic drivers of MishaYT (from here https://github.com/MishaTY/Misha-OS) and I use vmware player 16
do you know how to do it ?
The issue : https://github.com/CosmosOS/Cosmos/issues/1684
Thanks

**Comment by MishaTy** [comment]
I have fixed the crashing issue with a new video driver. Redownload Misha OS to get the new video driver.

**Reply by RetroAndDev** [reply]
Ok but how i can add other resolution (like 800x600) ?

**Comment by quajak** [comment]
The problem with the video memory is most likely a problem with cosmos memory management, which will be worked on in the near future. "Emptying" the video memory therefore wouldnt work.

**Reply by RetroAndDev** [reply]
Ok.
Thaks 
I'm trying to implement the MishaOS drivers on my OS.
I'll get back to you as soon as I'm done

**Reply by MishaTy** [reply]
The new driver that I used is: https://github.com/napalmtorch/CosmosVGA

**Reply by quajak** [reply]
If you are on the devkit, you can try use the double buffered standard drivers, which I hope shouldn't freeze

**Reply by RetroAndDev** [reply]
VGA ?

**Reply by quajak** [reply]
AFAIK all three should have a double buffered version.

**Reply by RetroAndDev** [reply]
AFAIK ?
